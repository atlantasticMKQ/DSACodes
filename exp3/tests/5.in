{{},49,{{{},67,{{},76,{{},50,{{{},18,{}},55,{{{{{},30,{{},65,{{{},40,{}},100,{{{{},63,{}},26,{{{},7,{{},15,{}}},54,{{},42,{{{{},72,{{{{},24,{}},11,{{},29,{}}},35,{}}},28,{}},90,{{},43,{{{},78,{}},8,{{{},88,{{},81,{{},33,{{},70,{}}}}},14,{{{{{},60,{}},37,{}},31,{}},66,{{{},25,{{},17,{}}},47,{}}}}}}}}}},95,{{{{{{{{},57,{{{},12,{}},38,{{{{},77,{}},51,{{},19,{}}},48,{}}}},73,{{},21,{{{{{},69,{}},84,{}},2,{{{},1,{}},91,{{},80,{}}}},52,{{{{},41,{}},53,{{},92,{}}},71,{}}}}},97,{{{},64,{}},75,{{{{},3,{}},98,{{},46,{}}},89,{}}}},10,{{{{},62,{}},96,{{},34,{}}},79,{{{},27,{}},85,{{},22,{}}}}},56,{}},83,{}},87,{{},44,{}}}}}}},99,{{},39,{{{{},94,{{},86,{{},36,{}}}},45,{{{{},61,{}},59,{{},93,{}}},5,{{{},13,{}},16,{{},20,{}}}}},9,{}}}},23,{}},68,{{},32,{{{},82,{}},4,{{},6,{}}}}}}}}},58,{{},74,{}}}}
