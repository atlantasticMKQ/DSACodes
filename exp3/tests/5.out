49,58,67,76,50,55,18,68,23,99,30,65,100,40,95,26,63,54,7,15,42,90,28,72,35,11,24,29,43,8,78,14,88,81,33,70,66,31,37,60,47,25,17,87,83,56,10,97,73,57,38,12,48,51,77,19,21,52,2,84,69,91,1,80,71,53,41,92,75,64,89,98,3,46,79,96,62,34,85,27,22,44,39,9,45,94,86,36,5,59,61,93,16,13,20,32,4,82,6,74
49,58,67,74,76,50,55,18,68,23,32,99,4,30,39,82,6,65,9,100,45,40,95,94,5,26,87,86,59,16,63,54,83,44,36,61,93,13,20,7,42,56,15,90,10,28,43,97,79,72,8,73,75,96,85,35,78,14,57,21,64,89,62,34,27,22,11,88,66,38,52,98,24,29,81,31,47,12,48,2,71,3,46,33,37,25,51,84,91,53,70,60,17,77,19,69,1,80,41,92
22
11
24